cmd/eksctl-anywhere/cmd/installpackages.go

```diff
@@ -0,0 +1,104 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+
+	"github.com/aws/eks-anywhere/pkg/config"
+	"github.com/aws/eks-anywhere/pkg/curatedpackages"
+	"github.com/aws/eks-anywhere/pkg/kubeconfig"
+	"github.com/aws/eks-anywhere/pkg/version"
+)
+
+type installPackageOptions struct {
+	source      curatedpackages.BundleSource
+	kubeVersion string
+	name        string
+	registry    string
+}
+
+var ipo = &installPackageOptions{}
+
+func init() {
+	installCmd.AddCommand(installPackageCommand)
+	installPackageCommand.Flags().Var(&ipo.source, "source", "Location to find curated packages: (cluster, registry)")
+	if err := installPackageCommand.MarkFlagRequired("source"); err != nil {
+		log.Fatalf("Error marking flag as required: %v", err)
+	}
+	installPackageCommand.Flags().StringVar(&ipo.kubeVersion, "kubeversion", "", "Kubernetes Version of the cluster to be used. Format <major>.<minor>")
+	installPackageCommand.Flags().StringVar(&ipo.name, "name", "", "Custom name of the curated package to install")
+	if err := installPackageCommand.MarkFlagRequired("name"); err != nil {
+		log.Fatalf("Error marking flag as required: %v", err)
+	}
+}
+
+var installPackageCommand = &cobra.Command{
+	Use:          "packages [flags]",
+	Aliases:      []string{"package", "packages"},
```

Comment:
curious: what's the reason to allow `install packages` if users can only install one package?

Reply:
That's a good point. Let me change that.